src/test/scala/uk/gov/nationalarchives/tdr/api/service/ConsignmentServiceSpec.scala

```diff
@@ -88,4 +88,34 @@ class ConsignmentServiceSpec extends AnyFlatSpec with MockitoSugar with Matchers
 
     response should be(None)
   }
+
+  "consignmentHasFiles" should "return true when files already associated with provided consignment id" in {
+    val fixedUuidSource = new FixedUUIDSource()
+    val fileUuid1 = UUID.randomUUID()
+    val fileUuid2 = UUID.randomUUID()
+    val mockResponse: Future[Boolean] = Future.successful(true)
+    val consignmentRepoMock = mock[ConsignmentRepository]
+    when(consignmentRepoMock.consignmentHasFiles(any[UUID])).thenReturn(mockResponse)
```

Comment:
Lots of random UUIDs in this test. Is there any reason this can't check for the specific UUID rather than `any[UUID]`? It would make the test less likely to pass for the wrong reason, and it makes it easier to read too, because you can tell which of the three UUIDs are actually important.

Reply:
Removed the random UUID